Client/Frontend/Browser/BrowserViewController/BVC+Rewards.swift

```diff
@@ -93,7 +112,26 @@ extension BrowserViewController: RewardsDataSource {
         }
     }
     
-    func displayString(for url: URL) -> String? {
-        return url.host
+    func pageHTML(for tabId: UInt64, completionHandler: @escaping (String?) -> Void) {
```

Comment:
where does the result of this go? need to make sure it's not evaluated anywhere since page HTML is untrusted input

Reply:
The HTML is pushed through to the native ledger and ads library. 
For ads its for classifying the page: https://github.com/brave/brave-core/blob/5a217978eee7639e7d8b9b755a4af11504698dda/vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc#L554
For ledger its used for twitch.tv publishers to scrape usernames I believe: https://github.com/brave/brave-core/blob/5a217978eee7639e7d8b9b755a4af11504698dda/vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.cc#L1031

Reply:
got it